http/src/main/resources/reference.conf

```diff
@@ -386,6 +386,7 @@ azure {
    wds-app-config {
      environment = "dev"
      environment-base = "live"
+     image: "us-central1-docker.pkg.dev/dsp-artifact-registry/mf-dataplane-metrics-wip/wds:insights"
```

Comment:
Often to avoid the whole Leo dance I just `kubectl edit deploy` on a live WDS/CBAS/etc deployment to change the image and iterate.

Reply:
Thanks, yeah I know that's faster, wanted to test something end to end :)

Reply:
Also apparently I don't have kubectl access to landing zone aks clusters and I'm not sure how give that access to myself. I'm being blocked by deny assignments. I am in the `terra-workspace-dev` AD group